tech.json

```diff
@@ -470,12 +470,20 @@
         {
           "name": "canBombHorizontally",
           "requires": [],
-          "note": "A horizontal bomb jump using a single bomb. Since it uses one bomb, it works with PBs as well.",
+          "note": [
+            "Place a single bomb then move Samus to the side to be boosted away from the explosion.",
```

Comment:
I think it might be worthwhile to note that you often only need to barely tap the horizontal movement. I used to think you had to have really good timing because I didn't know this. 

It might also be worthwhile to note that it can be used at the end/top of an IBJ?